src/renderer/Presentational/Sidebar/sidebar.css.ts

```diff
@@ -6,14 +6,19 @@ export const container = style({
   flexDirection: 'column',
   alignItems: 'flex-start',
   boxSizing: 'border-box',
-  padding: '0px 16px 16px 16px',
+  padding: '0px 13px 16px 16px',
   paddingTop: '52px',
   isolation: 'isolate',
   width: '268px',
   height: '100%',
-  borderRight: vars.components.sidebarBorder,
   backgroundColor: vars.components.sidebarBackground,
   // backdropFilter: 'blur(40px)',
+  selectors: {
+    '&.darwin': {
+      backgroundColor: vars.components.sidebarMacBackground,
+      backdropFilter: 'blur(40px)',
```

Comment:
Just curious what this looks like. I don't know if I see this locally on my mac.

Reply:
Yeah, it's kind of hard to tell.. I set the transparency values as defined in the design 🤔

@ddannehh, this is what it looks like for dark and light mode. Do we want to make it a little more transparent, similar to the Finder app on mac? it doesn't have to happen in this PR, I can tweak it in another.

<img width="1053" alt="Screen Shot 2023-09-20 at 1 44 17 PM" src="https://github.com/NiceNode/nice-node/assets/99524051/32d7fa94-6b26-44a6-bf16-20cf2fe8c168">
<img width="1058" alt="Screen Shot 2023-09-20 at 1 44 08 PM" src="https://github.com/NiceNode/nice-node/assets/99524051/343e44d5-8f18-48a1-97b8-696a43201271">

Reply:
a nice touch!

Reply:
Looks okay-ish to me, but hard to tell without a Finder window -as reference- window next to it. Couldn't find what the exact formula is for replicating the native macOS behaviour with just CSS but after some research I believe it's impossible.
I've found some resources covering this and my understanding is that it's achievable through Electron configuration since "The CSS `blur()` filter only applies to the window's web contents, so there is no way to apply blur effect to the content below the window (i.e. other applications open on the user's system)." [source](https://www.electronjs.org/docs/latest/tutorial/window-customization#create-transparent-windows).
The right approach would be to set `win.setVibrancy(type)`[source](https://www.electronjs.org/docs/latest/api/browser-window#winsetvibrancytype-macos).

Reply:
Ah, I figured out a way to change the electron window appearance to override the system setting, and match the user setting:

```
function createWindow() {
  let winAppearance = 'dark';
  
  // Check system's current theme
  if (nativeTheme.shouldUseDarkColors) {
    winAppearance = 'light';
  }

  mainWindow = new BrowserWindow({
    width: 800,
    height: 600,
    webPreferences: {
      nodeIntegration: true,
    },
    appearance: winAppearance,
  });

  mainWindow.loadFile('index.html');

  // Listen for theme change from renderer
  ipcMain.on('change-theme', (event, theme) => {
    mainWindow.setAppearance(theme);
  });
}
```

If you're okay with the full native transparency, I can put this change in another PR 😄 @ddannehh

Reply:
I believe this is the way! I'm definitely okay with full native behaviour for the sidebar. 💯